jujugui/static/gui/src/app/components/charmbrowser/charmbrowser.js

```diff
@@ -86,7 +86,10 @@ YUI.add('charmbrowser-component', function() {
                 importBundleYAML={this.props.importBundleYAML}
                 getBundleYAML={this.props.getBundleYAML}
                 getEntity={this.props.getEntity}
+                getDiagramURL={this.props.getDiagramURL}
                 deployService={this.props.deployService}
+                getFile={this.props.getFile}
+                renderMarkdown={this.props.renderMarkdown}
```

Comment:
I think we could simplify the number of functions being passed in by moving some (particularly these) into the entity model; that is, the entity model would invoke the underlying functions in `charmstore-api.js`, saving us the hassle of passing them down to every component.